### PR TITLE
Create mkdocs.yml

### PR DESCRIPTION
**Pull request by absrocks**
Added mkdocks.yml

### CODE REVIEW

**Comment by alexpacheco** [comment]
@absrocks NJIT Red has a rgb value of 204,0,0 that converts to cc0000 in hex. The red used here, c44831, is not correct. However, the logo's background red is also not cc0000. Can you check the correct color of the logo's background and set that as the color for primary color. Also, you might want to use `--md-footer-bg-color` in stylesheets/extra.css to change the color of the footer

File: docs/Softwares/matlab.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# Use MATLAB on NJIT HPC
+
+## Installation steps of MATLAB on local machine
```
Comment (alexpacheco) [inline docs/Softwares/matlab.md:3]: Our software application pages should follow a template for consistency.

* short description of the application
* table containing the various versions and their module files (maybe a note column if a particular toolchain module needs to be loaded)
* how to use the application on our cluster
* example of a slurm script, example files or location on the cluster where the example files can be found.
* How to use application if available on Bright or Ondemand portal
* Any tips or tricks for using the application or external links that may be useful including tutorials, user guide from developer etc
* Lastly, how to use/install the application locally and connect from it to the cluster (for example this first section here should be last). 

@gwolosh anything you want to add?

**Review by alexpacheco — APPROVED** [review]
Looks good for 1st draft

**Comment by alexpacheco** [comment]
1st draft of website

File: docs/Softwares/matlab.md (hunk 1 above)
Comment (absrocks) [reply docs/Softwares/matlab.md:3]: Thanks @alexpacheco for the suggestions. I see this pull request is closed, however, the files are not updated in the "main" branch. The status is "Closed with unmerged commits".

**Comment by alexpacheco** [comment]
Not sure why merge didn't work ... Reopening

**Pull request merged**